pkg/notifiers/prune.go

```diff
@@ -58,18 +57,18 @@ func filterProcesses(processes []*storage.ProcessIndicator, maxSize int, currSiz
 		return processes
 	}
 
-	marshaler := new(jsonpb.Marshaler)
+	marshaler := new(protojson.MarshalOptions)
 	// Clean Process first then prune
 	for _, p := range processes {
 		cleanProcessIndicator(p)
 	}
 
 	for i := len(processes) - 1; i >= 0; i-- {
-		var data bytes.Buffer
-		if err := marshaler.Marshal(&data, processes[i]); err != nil {
+		data, err := marshaler.Marshal(processes[i])
```

Comment:
How about a `jsonutil.MarshalToCompactString` function ? That would avoid having a mix of direct calls to `protojson` and calls to `protojson` through `jsonutil`.